Sanity/main.fmf

```diff
@@ -7,6 +7,8 @@ test: ./runtest.sh
 recommend:
   - tang
   - clevis
+require:
+  - jq
```

Comment:
May I ask to also require helm and podman, so that non-functional tests are also executed?

Reply:
Sure, I'll add there these requires.